lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -165,7 +165,7 @@ def setup(self):
                 self.use_energy_dependent_cuts = (
                     "GH_CUT" not in hdul["EFFECTIVE AREA"].header
                 )
-        except KeyError:
+        except:
             raise ToolConfigurationError(
```

Comment:
The error message here is misleading. As said, the code above will only raise an error if:

* The input irf file does not exist (which I think is checked before)
* The irf file does not have an `EFFECTIVE AREA` HDU
Nothing to do with a `GH CUTS` table